CheckDrive.Api/CheckDrive.Infrastructure/Helpers/JwtTokenGenerator.cs

```diff
@@ -0,0 +1,59 @@
+﻿using CheckDrive.Application.Interfaces.Authorization;
+using CheckDrive.Infrastructure.Configurations;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.Extensions.Options;
+using Microsoft.IdentityModel.Tokens;
+using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using System.Text;
+
+namespace CheckDrive.Infrastructure.Helpers;
+
+public sealed class JwtTokenGenerator : IJwtTokenGenerator
+{
+    private readonly JwtOptions _options;
+
+    public JwtTokenGenerator(IOptions<JwtOptions> options )
+    {
+        _options = options.Value ?? throw new ArgumentNullException(nameof(options));
```

Comment:
You should check for null before accessing the object, otherwise you'll get nullreferenceexception

Reply:
> You should check for null before accessing the object, otherwise you'll get nullreferenceexception

I dont understand please explain more details there